app/views/home.njk

```diff
@@ -13,7 +13,7 @@
           <p class="govuk-body">Reductions to delinked payments are known as progressive reductions.</p>
           <p class="govuk-body">To receive delinked payments, you must have claimed, and been eligible for, BPS 2023 in England (except in some inheritance cases).</p>
         </br>
-        <p class="govuk-body">Use this calculator to estimate how progressive reductions could reduce your delinked payment for 2025. The estimate for 2025 is based on the proposed progressive reduction figures for 2025 shown in the delinked payments guidance. The calculator also shows how progressive reductions affected your 2024 delinked payment.</p>
+        <p class="govuk-body">Use this calculator to estimate how progressive reductions could reduce your delinked payment for 2025. The estimate for 2025 is based on the proposed progressive reduction figures for 2025 shown in the <a class="govuk-link" rel="external" href="https://www.gov.uk/guidance/delinked-payments-replacing-the-basic-payment-scheme"> delinked payments guidance</a>. The calculator also shows how progressive reductions affected your 2024 delinked payment.</p>
```

Comment:
don't think you need a space within your hyperlink as you have one before - otherwise comes through as a double space?

Reply:
discussed

Reply:
No need, it's only single spaced